R/case.R

```diff
@@ -64,7 +64,7 @@ vt_use_test_case <- function(name, username = vt_username(), title = NULL, open
   }
 
   # Add file to validation configuration
-  vt_add_file_to_config(filename = name, after = add_after, before = add_before)
```

Comment:
Kudos to you. I was playing around with the curly-curly but it was giving me trouble and just relied on lazy evaluation instead.